source/exe/win32/platform_checks.cc

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "envoy/filesystem/filesystem.h"
+
+namespace Envoy {
+
+// Check system wide configurations such as user level
+// file watches, file limit permissions etc...
+// TODO(windows maintainers)
+void checkPlatformSettings(Filesystem::Instance&) {
+  ENVOY_LOG_MISC(
+      warn,
```

Comment:
I think I mentioned this before, but i think this will be too noisy for Windows users. A TODO here should be fine (maybe an open bug too).